lib/actors/actor/interface/http.ex

```diff
@@ -72,17 +71,17 @@ defmodule Actors.Actor.Interface.Http do
   defp do_invoke_host(payload, state) do
     payload
     |> ActorInvocation.encode()
-    |> Client.invoke_host_actor()
+    |> @http_node_client.invoke_host_actor()
```

Comment:
In this case you switched to doing this at compile time, this would prevent us from trying a Daemonsets based approach instead of just Sidecars in the future. But we can change this later and so we will continue this way for now.